docs/standard/base-types/enumeration-format-strings.md

```diff
@@ -26,13 +26,13 @@ You can use the <xref:System.Enum.ToString%2A?displayProperty=fullName> method t
 |Format string|Result|  
 |-------------------|------------|  
 |G or g|Displays the enumeration entry as a string value, if possible, and otherwise displays the integer value of the current instance. If the enumeration is defined with the **Flags** attribute set, the string values of each valid entry are concatenated together, separated by commas. If the **Flags** attribute is not set, an invalid value is displayed as a numeric entry. The following example illustrates the G format specifier.<br /><br /> [!code-csharp[Formatting.Enum#1](../../../samples/snippets/csharp/VS_Snippets_CLR/Formatting.Enum/cs/enum1.cs#1)]
- [!code-vb[Formatting.Enum#1](../../../samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb#1)]|  
+| |[!code-vb[Formatting.Enum#1](../../../samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb#1)]|
```

Comment:
@ASchwenkerBM Thanks for working on this rather badly broken table. :+1: This commit puts the two language samples on different rows ...

![capture](https://user-images.githubusercontent.com/1622880/27008166-3f1deea2-4e30-11e7-91d6-2754598e3d01.PNG)

How about we move the two samples to the same row by putting a `<br>` between the two samples? For example using this first row where this occurs ...

> |G or g|Displays the enumeration entry as a string value, if possible, and otherwise displays the integer value of the current instance. If the enumeration is defined with the **Flags** attribute set, the string values of each valid entry are concatenated together, separated by commas. If the **Flags** attribute is not set, an invalid value is displayed as a numeric entry. The following example illustrates the G format specifier.\<br>\<br>\[!code-csharp\[Formatting.Enum#1](../../../samples/snippets/csharp/VS_Snippets_CLR/Formatting.Enum/cs/enum1.cs#1)]\<br>\[!code-vb\[Formatting.Enum#1](../../../samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb#1)]|

I suggest just one \<br> to start. Since the review site isn't publicly available after the build on the next commit, I'll post back how it renders with one \<br> tag and then we can decide if it might render better with two \<br> tags if we don't like what we see.